template/src/main.js

```diff
@@ -617,6 +612,21 @@ function init () {
     $('.sidenav').find('a.nav-list-item').show();
   });
 
+  /**
+   * Executing a function after a specified amount of time
+   *
+   * @param {*} fn function to call after ms
+   * @param {*} ms ms to wait before callback
```

Comment:
Why are you not describing the function parameters more precisely? `fn` should be a function/callable (whatever works for jsdoc) and `ms` would be better called `timeDelay` and its type should be number. In fact it would be best to have the second argement called `delay` so it's the same name as in `setTimeout`, but it means the function should be renamed and I don't have a good suggestion. Maybe `temporize`?

Reply:
My suggestion would be to call the function `debounce` and the parameters `callback` and `delay`

Reply:
Agree for the parameters name, but debounce? Sounds weird to me.

Reply:
Know comment looks like this:
```
/**
 * Executing the callback after the specified delay.
 * Resets the timer if called again before the delay is reached.
 *
 * Behavior to prevent too many events from being triggered and getting stuck.
 *
 * @param {*} callback function to call after delay expires.
 * @param {*} delay the time, in milliseconds that the timer should wait before the specified function or code is executed.
 * @returns Timeout function includes the callback
 */
```